test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -28,4 +28,24 @@
 
 #include "session_simulator.h"
 
+#include <iostream>
+
 session_simulator::session_simulator() {}
+
+bool
+session_simulator::begin_transaction()
+{
+    /*
+     * Cannot begin a new transaction if there is an existing transaction that is currently running.
```

Comment:
I think `s_all` did this (multi-line comment)?

Reply:
The comment tags don't fit on the same line.

Reply:
Yea, best to make the comment multi-line then.